backends-velox/src/test/scala/io/glutenproject/execution/VeloxFunctionsValidateSuite.scala

```diff
@@ -513,6 +513,31 @@ class VeloxFunctionsValidateSuite extends VeloxWholeStageTransformerSuite {
     }
   }
 
+  test("Test make_ym_interval function") {
+    withTempPath {
+      path =>
+        Seq[Tuple2[Integer, Integer]]((1, 0), (-1, 1), (null, 1), (1, null))
+          .toDF("year", "month")
+          .write
+          .parquet(path.getCanonicalPath)
+
+        spark.read.parquet(path.getCanonicalPath).createOrReplaceTempView("make_ym_interval_tbl")
+
+        runQueryAndCompare("select make_ym_interval(year, month) from make_ym_interval_tbl") {
+          checkOperatorMatch[ProjectExecTransformer]
+        }
+
+        runQueryAndCompare("select make_ym_interval(year) from make_ym_interval_tbl") {
+          checkOperatorMatch[ProjectExecTransformer]
+        }
+
+        // select make_ym_interval() from make_ym_interval_tbl will be optimized with fallback.
```

Comment:
Maybe, need to add some handling in `FallbackEmptySchemaRelation` to allow offloading this.